### DISCUSSION TITLE
Do you have a Github Sponsor link?

### DISCUSSION THREAD

**Discussion by marcfalk** [airjp73/rvf#67] [Q&A] [answered]
This library has saved me quite some time. Thanks for that 🙏

I'd be happy to buy you a week's worth of coffee. Do you have [a button anywhere](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-user-account) for that? 😄

Keep it up!

**Comment by airjp73** [comment, marked as answer]
You can do that [here](https://github.com/sponsors/airjp73). 😄
